server/container/guice/cassandra-guice/src/test/java/org/apache/james/JamesWithNonCompatibleElasticSearchServerTest.java

```diff
@@ -58,7 +60,14 @@ static void afterAll() {
     @Test
     void jamesShouldStopWhenStartingWithANonCompatibleElasticSearchServer(GuiceJamesServer server) throws Exception {
         assertThatThrownBy(server::start)
-            .isInstanceOf(ElasticsearchStatusException.class);
+            .isInstanceOfSatisfying(
+                StartUpChecksPerformer.StartUpChecksException.class,
+                ex -> assertThat(ex.getBadChecks())
+                    .contains(CheckResult.builder()
+                        .checkName(ElasticSearchStartUpCheck.CHECK_NAME)
+                        .resultType(StartUpCheck.ResultType.BAD)
+                        .description(ElasticSearchStartUpCheck.notCompatibleWith(dockerES2.esVersion()))
```

Comment:
Write the plain String here please - that would be waay simpler to write and understand.